tools/DebianPackageTool/package_tool

```diff
@@ -216,6 +219,12 @@ package_docs(){
     fi
 }
 
+package_debian(){
+    if [[ -d "$ABSOLUTE_PLACEMENT_DIR" ]]; then
```

Comment:
To follow the design I mentioned above, this would copy all files from `$INPUT_DIR/debian` to `$DEBIAN_DIR`

That would allow for any kind of script or custom debian metadata.